src/test/java/com/google/api/codegen/testdata/php_main_library.baseline

```diff
@@ -1207,7 +1213,7 @@ class LibraryServiceApi
      * try {
      *     $libraryServiceApi = new LibraryServiceApi();
      *     $formattedName = LibraryServiceApi::formatBookName("[SHELF]", "[BOOK]");
-     *     $indexName = "";
+     *     $indexName = "default index";
```

Comment:
It looks like a feature that was added to toolkit but either wasn't implemented or wasn't correctly in PHP:

https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/testdata/library_gapic.yaml#L308